tf2onnx/onnx_opset/nn.py

```diff
@@ -571,10 +572,16 @@ def version_1(cls, ctx, node, **kwargs):
                     neg_diff_d = ctx.make_node("Neg", [diff_d.output[0]])
                     shrink_d_by = ctx.make_node("Max", [neg_diff_d.output[0], const_zero.output[0]])
                     sdb = shrink_d_by.output[0]
-                    pads = ctx.make_node("Concat", [cz, cz, cz, cz, cz, cz, shb, swb, sdb, cz], attr={"axis": 0})
+                    if is_channels_last(node):
+                        pads = ctx.make_node("Concat", [cz, cz, cz, cz, cz, cz, shb, swb, sdb, cz], attr={"axis": 0})
+                    else:
+                        pads = ctx.make_node("Concat", [cz, cz, cz, cz, cz, cz, cz, shb, swb, sdb], attr={"axis": 0})
```

Comment:
Could you mind add an unit test to test this new case?
May add a test in here? https://github.com/onnx/tensorflow-onnx/blob/a1a9343cef193ef45b7f91d511e9c7bd1dffb3e3/tests/test_backend.py#L5057

Reply:
AFAICT there are currently no tests for channels_first at all and I am unsure if I should add one: as @TomWildenhain-Microsoft mentioned in the issue, many TF ops have no CPU implementation for channels_first.

Reply:
Yes, if you do add a test, it won't run on the CI. You could still add one. I'm not on that team anymore so it is @hwangdeyu's call. The code looks correct to me and the unit tests at least ensure that you didn't break the existing CPU path. One of your lines is still too long for pylint btw.

Reply:
Thanks @TomWildenhain-Microsoft for reviewing it, I will finish ramping up as quickly as possible. But the way, it's possible to support skip the cpu tests by using any python decorator?
I guess it's good to miss test in this case.

Reply:
Decorators for skipping tests are defined here: https://github.com/onnx/tensorflow-onnx/blob/master/tests/common.py